wagtail_footnotes/blocks.py

```diff
@@ -53,7 +53,7 @@ def replace_tag(match):
         return mark_safe(FIND_FOOTNOTE_TAG.sub(replace_tag, html))  # noqa: S308
 
     def render(self, value, context=None):
-        if not self.get_template(context=context):
+        if not self.get_template(value=value, context=context):
```

Comment:
I'm pretty sure this was only added in 5.2

See https://github.com/wagtail/wagtail/blob/stable/5.1.x/wagtail/blocks/base.py#L210 vs https://github.com/wagtail/wagtail/blob/stable/5.2.x/wagtail/blocks/base.py#L213

Reply:
@zerolab ,
Thanks, I forgot to put the upgrade consideration under a conditional.
Please see [latest commit](wagtail_footnotes/blocks.py).